app/src/pages/AdminMenu/AdminMenuPage.tsx

```diff
@@ -1,15 +1,15 @@
-// AdminMenuPage.tsx
-import React from 'react';
+import React, { useState } from 'react';
 import { Navbar } from '../../components/Navigation/Navbar/Navbar';
 import { MenuIcon } from '../../components/Icons/MenuIcon';
 import { NavbarMiddle } from '../../components/Navigation/NavbarMiddle/NavbarMiddle';
-import { IconButton } from '../../components/IconButton/IconButton';
-import { BinIcon } from '../../components/Icons/BinIcon';
-import { EditIcon } from '../../components/Icons/EditIcon';
-import { Button } from '../../components/Button/Button';
-import { TableRow } from '../../components/Table';
+import { CategoryTable } from './components/CategoryTable';
+import { MenuItemsTable } from './components/MenuItemsTable';
 
 export const AdminMenuPage = () => {
+  const [selectedCategory, setSelectedCategory] = useState<{
+    name: string;
+  } | null>(null);
+
   const categories = [
```

Comment:
Can the categories and menu items be passed to this AdminMenuPage component as props. Then update the story by passing in the props as arguments